tests/test_models_unet.py

```diff
@@ -291,3 +292,7 @@ def test_output_pretrained_ve_large(self):
         # fmt: on
 
         self.assertTrue(torch.allclose(output_slice, expected_output_slice, rtol=1e-2))
+
+    def test_forward_with_norm_groups(self):
```

Comment:
Why?

Reply:
It seems to adjust norm groups inside the blocks  cf https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/unet_blocks.py#L780
So not sure if it can work with any value because of the division with 4